application-development/helloworld-springboot-auth/.classpath

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can we remove this file? 

I don't think we should be committing any IDE configuration files.